content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -15,7 +15,7 @@ Kubernetes {{< skew currentVersion >}} supports [Container Network Interface](ht
 (CNI) plugins for cluster networking. You must use a CNI plugin that is compatible with your cluster and that suits your needs. Different plugins are available (both open- and closed- source) in the wider Kubernetes ecosystem.
 
 You must use a CNI plugin that is compatible with the
-[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) release of the CNI specification.
+[v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md) release of the CNI specification.
```

Comment:
suggest 

```
You must use a CNI plugin that is compatible with the [v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) or later releases of the CNI specification. The current version of CNI is [v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md) release of the CNI specification.
```

Reply:
is `v1.0.0` still correct, or should it be `v1.1.0` now?

Reply:
Added this suggestion and squashed commits